lib/QuipProcessor.js

```diff
@@ -295,8 +295,27 @@ class QuipProcessor {
                 dateStr = updatedDate === createdDate? createdDate : `${createdDate} (Updated: ${updatedDate})`;
                 dateStr = `, ${dateStr}`;
             }
+
+            let referencedSections = "";
+            if(message.annotation && message.annotation.id) {
+                if(message.annotation.highlight_section_ids) {
+                    referencedSections = message.annotation.highlight_section_ids.map((sectionId, index) => {
+                        // Not sure if there can be more than one highlight section,
+                        // but if so, we can number the reference links
+                        const suffix =
```

Comment:
This formatting is due to [prettier](https://prettier.io/), but I did notice that it doesn't seem like there's a line length limit being enforced in this codebase, so let me know if you'd prefer this to be consolidated onto one line. Same goes for the other code in this patch!